debian/autobake-deb.sh

```diff
@@ -131,6 +138,7 @@ in
   "focal")
     replace_uring_with_aio
     disable_libfmt
+    remove_package_notes
```

Comment:
why do you remove package notes in focal and bullseye? too old binutils?

Reply:
yes, but more directly not a `dh-package-notes` package that would use it.
- https://packages.debian.org/bookworm/dh-package-notes
- https://packages.ubuntu.com/jammy/dh-package-notes